bundles/org.openhab.binding.onkyo/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -107,6 +107,11 @@
 		<label>Play URI</label>
 		<description>Plays a given URI</description>
 	</channel-type>
+	<channel-type id="audioinfo" advanced="true">
+		<item-type>String</item-type>
+		<label>Audio Info</label>
+		<description>Audio info - refresh timer must be set for updates</description>
```

Comment:
Since the refresh time must be configure for all channels to refresh, it might be better to describe that in the readme.

Reply:
Got it updated. Thanks!